_docs/guides/best-practices.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Best Practices 
+order: 004
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
+you fulfill the security best practice of least privilege. Following are the
+scenarios for which it's best to use separate service accounts:
+
+ * **[Forseti Security Service Account](#forseti-security-service-account)**
+ (required): Used by all core modules of the program to provide basic
+ inventory, scanning, and enforcement actions.
+ * **[GSuite Groups Service Account](#gsuite-groups-service-account)**
+ (optional): Used to inventory GSuite Groups and their members.
+ Forseti IAM explain requires this to be enabled.
+ * **[Forseti Explain Service Account](#forseti-explain-service-account)**
+ (optional): Used to provide IAM explain functionality.
+
+When you name service accounts, it's best to use a descriptive name like
+`forseti-security` or `forseti-security-gsuite`.
+
+### Forseti Security Service Account
```

Comment:
Out of scope for this change but I'd encourage you to rethink if this authority can be combined in a single service account. If what we want to achieve is least privilege, having less service accounts is almost as important to granting the 'boxes' less privileges.